src/core/packages/saved-objects/server-internal/src/routes/find.ts

```diff
@@ -62,6 +62,7 @@ export const registerFindRoute = (
             schema.oneOf([schema.string(), schema.arrayOf(schema.string())])
           ),
           sort_field: schema.maybe(schema.string()),
+          sort_order: schema.maybe(schema.oneOf([schema.literal('asc'), schema.literal('desc')])),
```

Comment:
Nit: this route is deprecated, IMO we should avoid adding new functionality to it, but I understand it is helpful for testing from an API perspective. Just might be better to test from an `soClient` perspective as this should be the primary way developers interface with this sort capability.